src/App.tsx

```diff
@@ -94,7 +94,9 @@ const App = () => {
 		flowId === 'saml-config' ||
 		flowId === 'sso-config';
 
-	const containerClasses = clsx('descope-base-container', {
+	const shadow = urlParams.get('shadow') !== 'false';
+
+	const containerClasses = clsx(shadow && 'descope-base-container', {
```

Comment:
I suggest that we put the class inside the object (like the other classes) to keep consistency